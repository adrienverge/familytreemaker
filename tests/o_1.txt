error: number of parents != 2
digraph {
	node [shape=box];
	edge [dir=none];

	LouisXV[label="test_1\n« Test_Surname»\n1710-02-15 † 1774-05-10",style=filled,fillcolor=azure2];

	{ rank=same;
	}
	{ rank=same;
	}
}
