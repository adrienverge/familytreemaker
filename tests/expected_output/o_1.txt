error: number of parents != 2
